universal-application-tool-0.0.1/app/services/applicant/ValidationErrorMessage.java

```diff
@@ -22,6 +22,8 @@
   public static String INVALID_ZIP = "Please enter valid 5-digit ZIP code.";
   public static String NO_PO_BOX = "Please enter a valid address. We do not accept PO Boxes.";
 
+  public static String ENTITY_NAME_REQUIRED = "Please enter a value for each entity.";
```

Comment:
suggestion (non-blocking): "entity" may be confusing for applicants - we can instead change this to the placeholder text , like `String.format("Please enter a value for each %s", placeholder)`

Reply:
dibe!